neo/Network/RPC/RpcServer.cs

```diff
@@ -361,6 +361,10 @@ public void Start(IPAddress bindAddress, int port, string sslCert = null, string
         {
             host = new WebHostBuilder().UseKestrel(options => options.Listen(bindAddress, port, listenOptions =>
             {
+                options.Limits.MaxConcurrentConnections = 40;
+                options.Limits.KeepAliveTimeout = TimeSpan.FromMinutes(1);
+                options.Limits.RequestHeadersTimeout = TimeSpan.FromSeconds(15);
+
```

Comment:
As the dns default resolving timeout is 15s, I think it's ok when we set the request timeout no less than 15s. 
By the way, could we set `options.Limits.MaxConcurrentConnections = Peers.MaxConnections`, as the seed node may set `MaxConnections` to 80.

Reply:
what strategic do you think that is better for ser a read timeout the read?

Reply:
It's better for the client to set the read timeout, but the server can set a maximum read timeout.

Reply:
We need it in the server side

Reply:
@Tommo-L limit with `Peers` looks reasonable as well.